test/eventing.bash

```diff
@@ -20,4 +20,8 @@ function upstream_knative_eventing_e2e {
   # run_conformance_tests defined in knative-eventing
   logger.info 'Starting eventing conformance tests'
   run_conformance_tests
+
+  # run_e2e_rekt_tests defined in knative-eventing
+  logger.info 'Starting eventing run_e2e_rekt_tests'
+  run_e2e_rekt_tests
```

Comment:
Hmm. This line adds a new test suite to the CI run. It should be sent separately in a different PR. We can then see how much more time it adds to CI execution. And we first need to discuss if we even want this test suite to be run here. I will start the discussion on our side. Please remove it for now from this PR. You can send it separately.

Reply:
yes this is removed and we will try to raised new PR against this